lib/dto/session.js.flow

```diff
@@ -0,0 +1,19 @@
+// @flow
+/**
+ * Flowtype definitions for session
+ * Generated by Flowgen from a Typescript Definition
+ * Flowgen v1.5.8
+ * Author: [Joar Wilk](http://twitter.com/joarwilk)
+ * Repo: http://github.com/joarwilk/flowgen
+ */
+
+export interface SessionDTO {
+  sessionId: string;
+  providerId: string;
+  providerCountry: string;
+  dateStarted: string;
+  bytesSent: number;
+  bytesReceived: number;
+  duration: number;
+}
+declare export function validateSession(data: any): SessionDTO;
```

Comment:
Pretty sure this DTO was changed to ConnectionSession so we probably don't need it.

@Waldz

Reply:
Yeah, we don't need it - this PR is already closed.